render-wasm/lint

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Enable debugging if the script is run with --debug
+if [[ "$1" == "--debug" ]]; then
+    set -x
+fi
+
+export SKIA_BINARIES_URL="https://github.com/penpot/skia-binaries/releases/download/0.80.1-1/skia-binaries-9e7d2684a17084095aef-wasm32-unknown-emscripten-gl-svg-textlayout.tar.gz"
```

Comment:
this url is also defined in the _build_env it would be great if we could define it in just one place

Reply:
sure! let me refactor this 👍🏽